requests_cache/core.py

```diff
@@ -159,9 +266,7 @@ def cache_disabled(self):
 
     def remove_expired_responses(self):
```

Comment:
Since `remove_old_entries()` takes a `datetime`, how about adding that as a parameter to `remove_expired_responses
()` as well?

Reply:
Actually, nevermind, that might just be confusing to users since `expired_before` is used to compare against `response.expires_after`, not replace it.